src/sortStudents.ts

```diff
@@ -1,16 +1,38 @@
 
+// describe Student interface
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
+
 }
 
+// describe SortType enum
 export enum SortType {
-  // describe SortType enum
+  name,
+  surname,
+  Age,
+  married,
+  averageGrade
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'desc' | 'asc';
+
+export function
+sortStudents(students: Student[], sortBy: SortType, order: SortOrder)
+  : Student[] |void {
```

Comment:
```suggestion
export function sortStudents(students: Student[], sortBy: SortType, order: SortOrder): Student[] {
```
there shouldn't be `void` return type